tutorial3/step2.md

```diff
@@ -1,12 +1,5 @@
-The vast majority of software packaged with unix and unix-like operating 
-systems comes with a **man page** - a manual to explain how to run the software.
-
-Now check out the [man page](https://en.wikipedia.org/wiki/Man_page) for `ls`.
```

Comment:
Do you think this link should be removed?

Reply:
Can you confirm this is resolved by 9aea226?